tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -178,6 +179,74 @@ public async Task CreateFromResourceResponseAsync_HttpClientFactory_Async()
             httpClientFactory.Received().GetHttpClient();
         }
 
+        [TestMethod]
+        public async Task CreateFromResourceResponseAsync_HttpClientFactory_Arm_TenantId_Async()

```

Comment:
Maybe can just add the `TenantId` assert to `CreateFromResourceResponseAsync_HttpClientFactory_Async`?

Reply:
This test just makes sure `httpClientFactory` is being called (and passed over). With the latest changes around `Func<HttpClient>` it became more clear.